build.xml

```diff
@@ -13,8 +13,8 @@
     <property name="build.dependencies" location="dependencies" />
 
     <path id="classpath">
-        <fileset dir="/usr/share/java" includes="**/*.jar" />
         <fileset dir="${build.dependencies}" includes="**/*.jar" />
+        <fileset dir="/usr/share/java" includes="**/*.jar" />
```

Comment:
Yes -- by re-ordering the dependencies here, the downloaded deps take precedence over any pre-installed deps on the system.